README.md

```diff
@@ -52,10 +52,10 @@ To ensure development libraries are installed, install everything in `requiremen
 
 ### Unit Tests
 
-Unit tests are in the `tests` folder. To run unit tests, use `unittest`:
+Unit tests are in the `tests` folder. To run unit tests, use the `test` script:
 
 ```
-> python -m unittest discover tests
+> ./scripts/test
```

Comment:
Moving to bash scripts in the instructions means Windows users won't have easy copy-paste instructions for working with the repo. Perhaps we could offer instructions for both ways of running tests and linting?

Reply:
I updated the `README` and Sphinx docs in e3dd0e3bf8c6a49edee691c71349ead0ed1c97f0.